src/components/Header/MobileNav.ts

```diff
@@ -263,7 +263,11 @@ export class MobileNav extends LitElement {
           ${unsafeHTML(open ? arrowDownIcon : arrowUpIcon)}
         </button>
         <ul class="items">
-          ${map(group.items, this.renderNavItem.bind(this))}
+          ${repeat(
+            group.items,
+            (item) => item.label,
+            this.renderNavItem.bind(this),
+          )}
```

Comment:
Zur Erklärung: Ich habe in den Templates von `map` auf `repeat` umgestellt, da `map` beliebig DOM-Nodes wiederverwendet, was mir Probleme bereitet hatte in den Tests. Das äusserte sich so, dass z.B. der Menü-Link «Angebote»  beim Querien durch Cypress so hiess (und also matchte), dann nach einem Re-rendering durch Lit jedoch plötzlich zum «Unterricht» Link wurde. Mit `repeat` bleiben nun die Elemente stabil (über den Key identifiziert).